README.md

```diff
@@ -1,2 +1,20 @@
 # pdffonts
-Node bindings for Poppler's pdffonts CLI
+
+Node bindings for Poppler's `pdffonts` CLI.
+
+## Dependencies
+
+For this module to install and build correctly, you'll need to make sure that [`poppler`](https://poppler.freedesktop.org/) is installed on your machine.
```

Comment:
that's what I did for calipers. https://github.com/calipersjs/calipers#-pdf-support
totally up to you though.